### PR TITLE
[1679] Backup Container

### PR DESCRIPTION
**Pull request by DerekRoberts**

### CODE REVIEW

**Comment by DerekRoberts** [comment]
Closing until ready.

File: openshift/backup.cj.yaml [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+            app: ${DATABASE_NAME}-backup
+            cronjob: ${DATABASE_NAME}-backup
+            template: "${JOB_NAME}-config-template"
+        spec:
+          backoffLimit: ${{JOB_BACKOFF_LIMIT}}
+          template:
+            metadata:
+              creationTimestamp: null
+            spec:
+              activeDeadlineSeconds: ${{JOB_DEADLINE_SECONDS}}
```
Comment (stephenhillier) [inline openshift/backup.cj.yaml]: Double checking here, are these variables meant to have double brackets?  There are a couple in this file.

File: openshift/backup.bc.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- name: GIT_REPO_URL
+  displayName: Git Repo URL
+  description: The URL to your GIT repo.
+  required: true
+  value: https://github.com/BCDevOps/backup-container.git
+- name: GIT_REF
+  displayName: Git Reference
+  description: The git reference or branch.
+  required: true
+  value: master
```
Comment (stephenhillier) [inline openshift/backup.bc.yaml]: we should use a commit here instead of master.

File: openshift/backup.cj.yaml (hunk 1 above)
Comment (DerekRoberts) [reply openshift/backup.cj.yaml]: Those are YAML variables required for properly handling integers.  They do look off though!

File: openshift/backup.bc.yaml (hunk 2 above)
Comment (DerekRoberts) [reply openshift/backup.bc.yaml]: Maybe!  I don't mind receiving new updates on master and can always fail back.

**Comment by DerekRoberts** [comment]
@stephenhillier Thanks!  There will be Jenkins changes soon, but I've been trying to break PRs up into smaller chunks.

**Comment by DerekRoberts** [comment]
[EPIC#Pipeline | Deployment and Cronjob database backup strategy |5| !!](https://trello.com/c/GOX9YEOs/1679-epicpipeline-deployment-and-cronjob-database-backup-strategy-5)

**Comment by DerekRoberts** [comment]
[EPIC#Pipeline | Deployment and Cronjob database backup strategy |5| !!](https://trello.com/c/GOX9YEOs/1679-epicpipeline-deployment-and-cronjob-database-backup-strategy-5)

**Pull request merged**